pkg/udf/function/uds_grpc_test.go

```diff
@@ -272,6 +272,240 @@ func TestHGRPCBasedUDF_ApplyWithMockClient(t *testing.T) {
 	assert.Equal(t, expectedKeys, resultKeys)
 }
 
+func TestGRPCBasedUDF_BasicApplyStreamWithMockClient(t *testing.T) {
```

Comment:
Since there is goroutines in these tests, I think we can apply the goleak library you found to ensure no leakage. I found it very useful. :)

```
func TestMain(m *testing.M) {
	goleak.VerifyTestMain(m)
}
```